core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1882,6 +1882,13 @@
 
   public static final String CARBON_MERGE_INDEX_IN_SEGMENT_DEFAULT = "true";
 
+  /**
+   * config carbon scan task locality
```

Comment:
Please provide more detail, like what scheduling behavior will be used for true and false